article_test.go

```diff
@@ -0,0 +1,71 @@
+package goshopify
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+)
+
+func TestBlogArticleList(t *testing.T) {
+	setup()
+	defer teardown()
+
+	httpmock.RegisterResponder(
+		"GET",
+		fmt.Sprintf("https://fooshop.myshopify.com/%s/blogs/1/articles.json", client.pathPrefix),
+		httpmock.NewStringResponder(
+			200,
+			`{"articles": [{"id": 1051293780,"title": "Welcome to the world of tomorrow!","created_at": "2013-11-06T19:00:00-05:00","body_html": "Good news, everybody!","blog_id": 241253187,"author": "dennis","user_id": null,"published_at": null,"updated_at": "2022-04-05T13:17:47-04:00","summary_html": null,"template_suffix": null,"handle": "welcome-to-the-world-of-tomorrow","tags": "","admin_graphql_api_id": "gid://shopify/OnlineStoreArticle/1051293780"}]}`,
+		),
+	)
+
+	options := ArticleListOptions{
+		ListOptions: ListOptions{Limit: 1},
+	}
+
+	articles, err := client.Article.ListBlog("1", options)
+	if err != nil {
+		panic(fmt.Sprintf("Cannot get blog list err: %s", err))
+	}
+
+	for _, article := range articles {
+		fmt.Printf("ID: %v\n", article.ID)
```

Comment:
Can you replace the Printf statements with a few checks that the strings match and fail if they're not?